ldk-server/Cargo.toml

```diff
@@ -18,3 +18,17 @@ hex = { package = "hex-conservative", version = "0.2.1", default-features = fals
 rusqlite = { version = "0.31.0", features = ["bundled"] }
 rand = { version = "0.8.5", default-features = false }
 async-trait = { version = "0.1.85", default-features = false }
+
+# Required for RabittMQ based EventPublisher. Only enabled for `events-rabbitmq` feature.
+lapin = { version = "2.5.1", optional = true }
```

Comment:
Looks like this is require a change of the MSRV to 1.81.0 with the feature enabled.